<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMeshFile</class>
 <widget class="QDialog" name="DialogMeshFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh file</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_translate_x">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_y">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_translate_y">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_scale">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_rotation_plan">
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_x">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_material">
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_material"/>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_x">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_translate_z">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_translate">
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_rotate_z">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_y">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale_z">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-9999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_path"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Path</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_path</tabstop>
  <tabstop>doubleSpinBox_translate_x</tabstop>
  <tabstop>doubleSpinBox_translate_y</tabstop>
  <tabstop>doubleSpinBox_translate_z</tabstop>
  <tabstop>doubleSpinBox_rotate_x</tabstop>
  <tabstop>doubleSpinBox_rotate_y</tabstop>
  <tabstop>doubleSpinBox_rotate_z</tabstop>
  <tabstop>doubleSpinBox_scale_x</tabstop>
  <tabstop>doubleSpinBox_scale_y</tabstop>
  <tabstop>doubleSpinBox_scale_z</tabstop>
  <tabstop>comboBox_material</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMeshFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMeshFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
